scripts/generate-app-config/generateRouteConfig.js

```diff
@@ -59,6 +77,13 @@ const contentRouteItem = (routePath, { contentPath, name, identifier }, props, t
     };
   }
 
+  if (type === 'evidence') {
+    contentProps = {
+      content: routeImporter.addImport(TerraScreenshotWrapper),
+      props: { imageConfig: evidenceProps(contentPath, routeImporter) },
```

Comment:
Should the `evidenceProps` helper create the `imageConfig` key in case there are changes to the Props rendered later on?